app/content/blog/wiki/how-to-update-ffxiv-price-data-on-universalis.md

```
@@ -0,0 +1,9 @@
+We get our data from [Universalis](https://universalis.app) and just changing prices does not update universalis.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)